github/doc.go

```diff
@@ -63,6 +63,21 @@ See the oauth2 docs for complete instructions on using that library.
 For API methods that require HTTP Basic Authentication, use the
 BasicAuthTransport.
 
+GitHub Apps authentication can be provided by
+https://github.com/bradleyfalzon/ghinstallation.
+
+	import "github.com/bradleyfalzon/ghinstallation"
+
+	func main() {
+		// Wrap the shared transport for use with the integration ID 1 authenticating with installation ID 99.
+		itr, err := ghinstallation.NewKeyFromFile(http.DefaultTransport, 1, 99, "2016-10-19.private-key.pem")
+		// Handle error.
+
+		// Use installation transport with client
+		client := github.NewClient(http.Client{Transport: itr})
+
+		// Use client...
+
```

Comment:
Okay, I see how this looks kinda bad/unbalanced without the closing `}` for `func main() {` now. So let's add that back in. Keep the `// Use client...` comment of course.